add aggregate endpoint and serializer

joncon opened the pull request.

ulbergc commented on app/measurement/views.py:
```diff
@@ -132,9 +136,11 @@ def get_queryset(self):
         return Metric.objects.all()
 
 
+REQUIRED_MEASUREMENT_PARAMS = ("metric", "starttime", "endtime")
```
Should add "channel" here I think. Definitely for the AggregateViewSet, double check for the MeasurementViewSet?

joncon commented:
channel is not required on measurement view since channel can be substituted for channel group id.  Removed constant and added channel to aggregated view
